angularSTEP/src/app/recipe-card/recipe-card.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { RecipeCardComponent } from './recipe-card.component';
```

Comment:
Like Chris said, we actually have started to move to not adding in spaces because GTS likes to remove them.